docs/readthedocs/source/_static/css/custom.css

```diff
@@ -101,4 +101,9 @@ footer.bd-footer{
 /* for tables in notebooks generated by pandas dataframe */
 .rendered_html table {
     table-layout: auto !important;
+}
+
+/* for right sidebar word-break */
+#bd-toc-nav ul li {
+    word-break: break-word;
```

Comment:
We are using `overflow-wrap: break-word;` for the middle contents. Is there a specific reason we use `word-break: break-word;` here?

If not, I feel like it is better to use `overflow-wrap: break-word;` here :D

Reply:
If `overflow-wrap: break-word;` does not take effect, then it LGTM.